src/facebook/facebook-notifications/facebooknotificationsyncadaptor.cpp

```diff
@@ -87,11 +87,17 @@ void FacebookNotificationSyncAdaptor::requestNotifications(int accountId, const
     queryItems.append(QPair<QString, QString>(QString(QLatin1String("locale")), QLocale::system().name()));
     QUrl url(graphAPI(QLatin1String("/me/notifications")));
     if (pagingToken.isEmpty()) {
-        int sinceSpan = m_accountSyncProfile
-                      ? m_accountSyncProfile->key(Buteo::KEY_SYNC_SINCE_DAYS_PAST, QStringLiteral("7")).toInt()
-                      : 7;
+        QDateTime since = lastSyncTimestamp(QLatin1String("facebook"),
```

Comment:
Uh, you're right, can't rely on lastSyncTimestamp as is. I suppose as a short term solution we could just store last successful sync as an account parameter?